app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -252,8 +293,76 @@ class CtaViewModel @Inject constructor(
                 }
             }
 
-            canShowPrivacyProCta() && extendedOnboardingFeatureToggles.privacyProCta().isEnabled() -> {
-                DaxBubbleCta.DaxPrivacyProCta(onboardingStore, appInstallStore)
+            canShowPrivacyProCta() -> {
+                if (highlightsOnboardingExperimentManager.isHighlightsEnabled()) {
+                    when {
```

Comment:
nit: I'd probably simplify all this and depending on the when get the title and description and then instantiate the Cta rather than copy&pasting 4 times the instantiation code.

Reply:
I agree. I might have misunderstood our conversation when implementing the experiment cohorts. I thought you suggested doing it this way instead of just checking cohorts to set resources.
I’ll simplified the code, let me know if that works for you